packages/domain/lib/entities/validation_entity.dart

```diff
@@ -0,0 +1,21 @@
+enum LoginValidationStatus {
+  ok,
+  empty,
+  notCorrect,
+}
+
+enum PasswordValidationStatus {
+  ok,
+  empty,
+  notCorrect,
+}
+
+class ValidationEntity {
```

Comment:
rename it to e.g LoginValidationException

Reply:
Fixed.